.travis.yml

```diff
@@ -7,6 +7,23 @@ matrix:
   include:
   - jdk: oraclejdk7
     script: mvn -q deploy --settings .travis-settings.xml -Dno.gem.deploy=true
+  - jdk: oraclejdk7
+    env: ANDROID_EXAMPLES=true
+    language: android
+    android:
+      components:
+        - build-tools-21.1.1
+        - platform-tools
+        - extra-android-support
+        - android-21
+        - sys-img-armeabi-v7a-android-21
+    script: mvn install -P android-examples -pl examples/android,examples/android/android-test -am -amd -Dno.gem.deploy=true -Dandroid.device=test
+    install: mvn install -P android-examples -pl examples/android,examples/android/android-test -am -amd -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
If there is no explicit `install`step, Travis will execute `mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V` as the `install` step, to make sure that the projects all dependencies have been installed with maven before the `script` command starts (if it detects that is is an maven project). Hm, now the [Travis docs](http://docs.travis-ci.com/user/languages/java/#Dependency-Management) specifies a slightly different command. 

In this case building (but not testing) all the Cucucmber-JVM modules (but not the Android examples), would be a lot of unnecessary work, so here I rather have the same command as the `script` command as `install` command - with the addition of `-DskipTests=true -Dmaven.javadoc.skip=true -B -V`. In this case `integration-test` or `verify` (should there be some extra checks in that step), should work just as well as `install` as the maven target (but why not use `mvn install`, when Travis would use it as default in the `install` step). I don't think `clean` adds anything as the whole job is executed in a fresh sandbox.